environment

```diff
@@ -5,5 +5,5 @@ export XSAMTOOLS_HOME="$(cd -P "$(dirname "$SOURCE")" && pwd)"
 
 set -a
 XSAMTOOLS_DOCKER_USER="xsamtools"  # name of user created in Dockerfile
-XSAMTOOLS_IMAGE_NAME="xbrianh/xsamtools"
+XSAMTOOLS_IMAGE_BASE_NAME="xbrianh/xsamtools"
```

Comment:
We should consider hosting this in a UCSC account.  Maybe https://quay.io/organization/ucsc_cgl ?

Reply:
I'll make an issue to track it.